# Monthly ga ge boo

Home expenses: 
- HOA: 160 
- home insurance: 70 
- property tax: 200

In [1]:
expense = {'home': 160 + 70 + 200}
expense

{'home': 430}

Utility fees
- electric: 100
- water: 100
- internet: 80
- phone: 30

In [2]:
expense['utility'] = 100 + 100
expense['internet'] = 80
expense['phone'] = 30
expense

{'home': 430, 'utility': 200, 'internet': 80, 'phone': 30}

Car expenses:
- insuarnce: 70
- gas: 30

In [3]:
expense['car'] = 100
expense

{'home': 430, 'utility': 200, 'internet': 80, 'phone': 30, 'car': 100}

In [4]:
def total():
    return sum(expense.values())

In [5]:
total()

840

So far, it is $840.

Grocery may take about $150 a week, which makes $600 a month.

In [6]:
expense['grocery'] = 600
expense

{'home': 430,
 'utility': 200,
 'internet': 80,
 'phone': 30,
 'car': 100,
 'grocery': 600}

In [7]:
total()

1440